services/scan/src/precinct_scanner_app.test.ts

```diff
@@ -302,15 +302,29 @@ test('ballot needs review - return', async () => {
   await waitForStatus(app, { state: 'needs_review', interpretation });
 
   await post(app, '/scanner/return');
-  await expectStatus(app, { state: 'returning', interpretation });
+  await expectStatus(app, {
+    state: 'returning',
+    interpretation,
+    canUnconfigure: false,
+  });
   await waitForStatus(app, {
     state: 'returned',
     interpretation,
-    ballotsCounted: 0,
+    canUnconfigure: false,
```

Comment:
I'm confused why the items in this struct are changing in the context of this PR?

Reply:
`waitForStatus` takes a partial expected status object and fills in the rest with defaults.

In this particular case `ballotsCounted: 0` is already the default, so I removed it for consistency and to make this less confusing overall (though I admit it's confusing if you don't know about the default thing).

 `canUnconfigure` has a default expected value in `waitForStatus` - `false` if `ballotCount == 0`, `true` otherwise. It's a crude approximation of the logic of `canUnconfigure`. I'm planning to move `canUnconfigure` out of this endpoint entirely because it's only used on the admin screen and isn't really part of the scanner status IMO, so I think it's fine to leave this a bit messy for now.